app/routes/_landing.($lang).docs/route.tsx

```diff
@@ -51,37 +59,79 @@ const organizeData = (docs: documentation[]): LinksGroupProps[] => {
       link: doc.slug || "",
       slug: doc.slug || "",
       links: findChildren(doc.id, docs),
+      hasParent: false,
     }))
 }
 
+function flattenTree(tree: LinksGroupProps[]) {
+  let result: LinksGroupProps[] = []
+
+  function dfs(node: LinksGroupProps) {
+    result.push(node)
+    for (let child of node.links) {
+      dfs(child)
+    }
+  }
+
+  for (let node of tree) {
+    dfs(node)
+  }
+
+  return result
+}
+
+function nextNodeInTree(nodes: LinksGroupProps[], node: LinksGroupProps | undefined) {
+  if (!node) return
+
+  const next = nodes.findIndex((n) => n.id === node.id)
+  if (next >= nodes.length - 1) return
+
+  return nodes[next + 1]
+}
+
 export default function DocsLayout() {
   const { data }: LoaderData = useLoaderData()
   const [linksGroupItems, setLinksGroupItems] = useState<LinksGroupProps[]>([])
   const organizeDataRef = useRef(organizeData)
+  const location = useLocation()
+
+  const flattenedTree = useMemo(() => flattenTree(linksGroupItems), [linksGroupItems])
+  const nextDoc = nextNodeInTree(
+    flattenedTree,
+    flattenedTree.find((ft) => location.pathname.includes(ft.slug)),
+  )
 
   useEffect(() => {
     if (data && data.length) {
-      console.log(data)
       const organizedData = organizeDataRef.current(data)
       setLinksGroupItems(organizedData)
     }
   }, [data])
 
   return (
-    <Grid
-      gutter="md"
-      sx={{
-        alignItems: "flex-start",
-        flexWrap: "nowrap",
-        justifyContent: "flex-start",
-      }}
-    >
-      {linksGroupItems && linksGroupItems.length ? (
-        <Sidebar data={linksGroupItems} />
-      ) : null}
-      <Box ml="56px">
+    <Grid gutter="sm">
+      <Grid.Col lg={2} md={12}>
+        {linksGroupItems && linksGroupItems.length ? (
+          <Sidebar data={linksGroupItems} />
+        ) : null}
+      </Grid.Col>
+      <Grid.Col lg={8} md={12}>
         <Outlet />
-      </Box>
+        {nextDoc && (
+          <Box
```

Comment:
We can start using Flex directly instead of Box with sx 

for example this can be replaced with 

`<Flex direction="column"> .... </Flex>`

Reply:
I mean your PR is the one updating the component library and this one is not based off your branch so

Reply:
I just used the one from mantine for now...